style.css

```diff
@@ -159,14 +159,14 @@ h1 {
  width: 75%;
  height: 30px;
  padding: 10px;
- color: #FF00FF;
+ color: var(--sharp-retroPink);
  position: relative;
  border-radius: 0px;
  font-family:'Gochi Hand', 'Just Another Hand', cursive;
  font-size: 16px;
  border-style: solid;
- border-color: #FF00FF;
- background-color: black;
+ border-color: var(--sharp-retroPink);
+ background-color: var(--onyx-black);
```

Comment:
Thanks for tweaking, loving the consistency 🥳

Reply:
🤩